test/PHPUnit/Controller/AbstractHttpControllerTestCaseTest.php

```diff
@@ -240,9 +241,17 @@ public function testAssertXpathQueryWithBadXpathUsage(): void
     {
         $this->dispatch('/tests');
 
-        $this->expectError();
+        // symfony/dom-crawler 5.4: InvalidArgumentException
+        // symfony/dom-crawler 6.0: ExpectationFailedException
 
-        $this->assertXpathQuery('form#myform');
+        try {
+            $this->assertXpathQuery('form#myform');
+        } catch (Throwable) {
+            $this->assertTrue(true);
```

Comment:
Better to assert the expected exception type here. Something like: 

```suggestion
        } catch (Throwable $e) {
            $this->assertTrue($e instanceof InvalidArgumentException || $e instanceof ExpectationFailedException);
```

Reply:
I've changed it to two catch blocks with message check